operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -190,7 +190,7 @@ Follow these steps when upgrading `ncn-m001`
 during [3.1 `management-nodes-rollout` with CSM upgrade](../workflows/management_rollout.md#31-management-nodes-rollout-with-csm-upgrade)
 when following the procedures in
 [Install or upgrade additional products with IUF](../workflows/install_or_upgrade_additional_products_with_iuf.md)
-or [Upgrade CSM and additional products](../workflows/upgrade_csm_and_additional_products.md).
+or [Upgrade CSM and additional products with IUF](../workflows/upgrade_csm_and_additional_products_with_IUF.md).
```

Comment:
IUF shouldn't be capitalized in the file name

```suggestion
or [Upgrade CSM and additional products with IUF](../workflows/upgrade_csm_and_additional_products_with_iuf.md).
```

Reply:
Fixed, thanks